OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/cg/AndroidICCAnalysis.scala

```diff
@@ -0,0 +1,1430 @@
+        val statements = tacCode.stmts
+        val addUseSites: ListBuffer[Int] = ListBuffer.empty[Int]
+
+        useSites.foreach { ln ⇒
+            val stm = statements(ln)
+
+            if (stm.isExprStmt && stm.asExprStmt.expr.isVirtualFunctionCall) {
+                    val virtualCall = stm.asExprStmt.expr.asVirtualFunctionCall
+                    val someIntent = internalFinder(virtualCall.name, virtualCall.params, intent, tacCode, virtualCall.pc)
+                    if (someIntent.isDefined) {
+                        val expIntent = someIntent.head
+                        expIntent.findStartedMethod(tacCode, useSites)
+                        if (expIntent.calledMethod.nonEmpty) {
+                            explicitIntents += expIntent
+                            return
+                        }
+                    }
+            } else if (stm.isVirtualMethodCall) {
+                val virtualCall = stm.asVirtualMethodCall
+                internalFinder(virtualCall.name, virtualCall.params, intent, tacCode, virtualCall.pc)
+            } else if (stm.isAssignment && stm.asAssignment.expr.isVirtualFunctionCall) {
+                    val virtualCall = stm.asAssignment.expr.asVirtualFunctionCall
+                    if (virtualCall.declaringClass == ObjectType(intentString)) { addUseSites ++= stm.asAssignment.targetVar.usedBy.toChain }
+                    val someIntent = internalFinder(virtualCall.name, virtualCall.params, intent, tacCode, virtualCall.pc)
+                    if (someIntent.isDefined) {
+                        val expIntent = someIntent.head
+                        expIntent.findStartedMethod(tacCode, useSites)
+                        if (expIntent.calledMethod.nonEmpty) {
+                            explicitIntents += expIntent
+                            return
+                        }
+                    } else if (stm.asAssignment.expr.isStaticFunctionCall) {
+                        val staticCall = stm.asAssignment.expr.asStaticFunctionCall
+                        if (staticCall.declaringClass == ObjectType(intentString)) addUseSites ++= stm.asAssignment.targetVar.usedBy.toChain
+                        internalFinder(staticCall.name, staticCall.params, intent, tacCode, staticCall.pc)
+                    }
+            } else if (stm.isInstanceOf[ReturnValue[DUVar[ValueInformation]]]) {
+                val intentTupel = intent.caller.classFile.thisType -> intent.caller.name
+                if (returningIntentsMap.contains(intentTupel)) {
+                    returningIntentsMap(intentTupel) += intent
+                } else returningIntentsMap += ((intent.caller.classFile.thisType -> intent.caller.name) -> ListBuffer(intent))
+            }
+        }
+        if (addUseSites.nonEmpty) {
+            evaluateUseSites(tacCode, addUseSites.toList, intent)
+        }
+        else implicitIntents += intent
+    }
+
+    def internalFinder(
+                        nme:       String,
```

Comment:
This is "name", right? If so, please write it out. No point in making things less clear for one saved letter.